### DISCUSSION TITLE
Need help in adding up PackageRules for Auto upgrading  image version mentioned in standalone .yaml file

### DISCUSSION THREAD

**Discussion by jadhavswapnil71** [renovatebot/renovate#25319] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.8.1

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Cloud (bitbucket.org)

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have self-hosted Renovate bot using K8 helm chart.
Attached file skaffold.yaml is only file in bitbucket repo.
I want to auto-update  the image of kaniko whenever new release is available .
[skaffold.txt](https://github.com/renovatebot/renovate/files/13053555/skaffold.txt)

Please help me in configuring renovate.json file which will be kept in the same repo at root level.
So my bot can pick it as per cron scheduled.

### What you tried so far.

{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    ":dependencyDashboard"
  ],
  "prConcurrentLimit": 100,
  "pinDigests": true,
  "packageRules": [
      {

        "matchUpdateTypes": ["minor", "patch", "digest"],
        "matchPaths": [
          "skaffold.yaml"
        ],
        "groupName": "skaffold-image-update",
        "matchPackageNames": ["gcr.io/kaniko-project/executor"],
  
      }
    ]
}

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
you need a regex manager
